src/main/archetype/ui.frontend/package.json

```diff
@@ -23,6 +23,7 @@
       "browserslist": "^4.2.1",
       "clean-webpack-plugin": "^3.0.0",
       "css-loader": "^3.0.0",
+      "copy-webpack-plugin": "^5.0.4",
```

Comment:
@thinkstainless - another picky one, can we list these alphabetically and keep the css related items together:

```
"clean-webpack-plugin": "^3.0.0",
"copy-webpack-plugin": "^5.0.4",
"css-loader": "^3.0.0",
"cssnano": "^4.1.10",
```